index.html

```diff
@@ -9,37 +9,213 @@
+            <div class="job-tech-lang">
+            <input type="button" value="html" class="job-item job-one"/>
+            <input type="button" value="bootstrap" class="job-item job-one"/>
+            <input type="button" value="Ruby" class="job-item job-one"/>
+            </div>
+        </div>
+        <button type="button" class="see-jobs-btn">See Project</button>
+        <div class="job-item-two">
+            <h1 class="job-post-title">Profesional Art Printing Data</h1>
+            <div class="job-post">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            </div>
+            <div class="job-tech-lang">
+            <input type="button" value="html" class="job-item job-one"/>
+            <input type="button" value="bootstrap" class="job-item job-one"/>
+            <input type="button" value="Ruby" class="job-item job-one"/>
+            </div>
+        </div>
+        <button type="button" class="see-jobs-btn">See Project</button>
+    </section>
+     <!-- About me section -->
+        <section class="about-me-section">
+            <div class="about-title">
+                <h1>About me</h1>
+            </div>
+            <div class="about-para">
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+            <div id="btn-line">
+                <button type="button" id="get-resume-btn">Get My Resume</button>
+            </div>
+    </section>
+    <section class="tech-list">
+            <div class="language">
+                <h1 class="language-header"><span><img src="./images/Rectangle-one.svg" alt=""></span>Language</h1>
+                <ul>
+                    <li>Javascript</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>Css</li>
+                </ul>
+            </div>
+            <div class="framework">
+                    <h1 class="framework-header"><span><img src="./images/Rectangle-two.svg" alt=""></span>Frameworks</h1>
+                    <ul>
+                        <li>Bootstrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSPec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
+                    </ul>
+            </div>
+            <div class="skills">
+                <h1 class="skills-header"><span><img src="./images/Rectangle-three.svg" alt=""></span>Skills</h1>
+                <ul>
+                    <li>Codekit</li>
+                    <li>Github</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
+        </div>
+    </section>
+    <!-- Get in touch section -->
```

Comment:
- [ ] Kindly let's only have one main container for the **About myself** section, for now, it seems that you are having two separate sections, but we can use only one `<section>` tag as the container of the **About myself** section and have other content inside as just `<div>` tags